sdks/python/apache_beam/io/lyft/s3.py

```diff
@@ -11,7 +11,7 @@
 from apache_beam.transforms.window import GlobalWindows
 
 Event = namedtuple('Event', 'name lateness_in_sec lookback_days')
-S3Config = namedtuple('S3Config', 'parallelism lookback_hours')
+S3Config = namedtuple('S3Config', 'parallelism lookback_hours  start_date end_date')
```

Comment:
I think it is better to use `start_timestamp_sec` and `end_timestamp_sec` so that it is clear the input is timestamp and provides more flexibility